### PR TITLE
Add fmt check

### PR DESCRIPTION
**Pull request by jichenjc**
command `make fmt` can be used to do format check now

**What this PR does / why we need it**:
add format check 
we can use ``make fmt`` to verify code format now
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
```

### CODE REVIEW

File: hack/verify-gofmt.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if ! which gofmt > /dev/null; then
+  echo "Can not find gofmt"
+  exit 1
+fi
+
+diff=$(find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -d 2>&1)
```
Comment (nikhita) [inline hack/verify-gofmt.sh]: This needs a `|| true` at the end.

Ref: https://github.com/kubernetes/kubernetes/blob/843a67b215dcfa1781820a797c4c759e42e81d3a/hack/verify-gofmt.sh#L52-L56

**Comment by jichenjc** [comment]
ok, updated the code

File: Makefile [hunk 2]
```diff
@@ -112,3 +112,6 @@ check-dependencies:
 	which hg >/dev/null 2>&1 || apt update && apt install -y mercurial
 	$$GOPATH/bin/dep version || (export DEP_RELEASE_TAG=v0.5.0; curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh)
 	dep status
+
+fmt:
+	hack/verify-gofmt.sh
```
Comment (alvaroaleman) [inline Makefile]: @jichenjc Can you please move that into the existing `lint` Maketarget?

Comment (nikhita) [reply Makefile]: How about renaming the `lint` target to `verify`?

Comment (alvaroaleman) [reply Makefile]: SGTM

**Comment by alvaroaleman** [comment]
We added a `go fmt` check as part of #426, hence this PR is not needed anymore

**Pull request closed without merging**